testing/src/main/java/org/oppia/testing/FakeExceptionLogger.kt

```diff
@@ -0,0 +1,22 @@
+package org.oppia.testing
+
+import org.oppia.util.logging.ExceptionLogger
+import javax.inject.Inject
+import javax.inject.Singleton
+import kotlin.Exception
+
+/** A test specific fake for the exception logger. */
+@Singleton
+class FakeExceptionLogger @Inject constructor() : ExceptionLogger {
+  var exceptionList = ArrayList<Exception>()
```

Comment:
A more Kotlin-idiomatic structure to use is a mutable list, e.g.:

```kotlin
private val exceptionList = mutableListOf<Exception>()
```

Also, prefer using ``val`` for read-only fields, and prefer to minimize access to members.

Reply:
Done.